### PR TITLE
onboard protein folding app

### PR DESCRIPTION
**Pull request by NiklasTR**

### CODE REVIEW

**Comment by thetechnocrat-dev** [comment]
give an example with flags once it is working.

**Comment by NiklasTR** [comment]
Run for quick view:
```
./plex -app colabfold-mini -input-dir testdata -gpu=true -network=true
```

**Review by thetechnocrat-dev — APPROVED** [review]
Ran on my macbook. Just one small change request.

Screen Shot 2023-03-10 at 11 11 58 AM

File: config/app.jsonl [hunk 1]
```diff
@@ -1,6 +1,12 @@
 {"app": "canarytest", "input-method": "index", "inputs": [{"field": "message", "filetypes": [".txt"]}, {"field": "csv", "filetypes": [".csv"]}], "outputs": [".txt"]}
 {"app": "diffdock", "input-method": "index", "inputs": [{"field": "protein", "filetypes": [".pdb"]}, {"field": "ligand", "filetypes": [".sdf", ".mol2"]}], "outputs": [".pdb", ".sdf", ".npy"]}
 {"app": "equibind", "input-method": "file", "inputs": [{"field": "protein", "filetypes": [".pdb"]}, {"field": "ligand", "filetypes": [".sdf", ".mol2"]}], "outputs": [".pdb", ".sdf"]}
-{"app": "equibindls", "input-method": "directory"}
+{"app": "_equibindls", "input-method": "directory", "inputs": [{"field": "protein", "filetypes": [".pdb"]}, {"field": "ligand", "filetypes": [".sdf", ".mol2"]}]}
```
[the remark this answers is not in the document]
Comment (NiklasTR) [reply config/app.jsonl:4]: sounds good!

**Pull request merged**